app/models/concerns/course/searchable_concern.rb

```diff
@@ -0,0 +1,18 @@
+module Course::SearchableConcern
```

Comment:
I'd call this SearchConcern, since the -able suffix is for polymorphic relations?